index.js

```diff
@@ -59,14 +59,47 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function() {
+    toggleColor(header)
+})
+
```

Comment:
great! I would like you to get used to using the event.target when working with event listeners. Great work passing in the `toggleColor` function.

great work! We have the header variable already defined earlier in the file.
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.

Great work incorporating the `toggleColor` function. We just need to pass in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```